modules/reference/pages/internal-metrics-reference.adoc

```diff
@@ -123,6 +129,34 @@ Number of Kafka RPC service errors.
 
 ---
 
+=== vectorized_ntp_archiver_log_compaction_bytes_removed
+
+Number of bytes removed from cloud storage by compaction operations. This is tracked on a per-topic and per-partition basis.
+
+This metric resets every time partition leadership changes. It primarily exists to track the fact that compaction is performing operations on cloud storage.
+
+The `namespace` label supports the following options for this metric:
+
+* `kafka` - user topics
+* `kafka_internal` - internal kafka topic such as consumer groups
+* `redpanda - redpanda-only internal data
+
+*Labels*:
+
+* `namespace=("kafka" | "kafka_internal" | "redpanda")`
+* `topic`
+* `partition`
+
+---
+
+=== vectorized_ntp_archiver_pending
+
+The offset of the last committed offset uploaded to tiered storage for each partition. This metric matching the last committed offset of a partition indicates that all data for a partition is uploaded to tiered storage.
+
+This metric is impacted by the `cloud_storage_upload_loop_max_backoff_ms` configuration property. If this metrics lags behind the latest committed offset by more than twice this property value, it may indicate a network issue in the upload path for the partition, or it may indicate a cluster that is under too much pressure.
+
```

Comment:
@Lazin , can you confirm?

Reply:
This metric is not an offset. It's an offset delta. Difference between the committed offset and last uploaded offset. Basically, it contains number of offsets that we ought to upload ASAP. You can also say that it's number of records which are not uploaded yet.

`cloud_storage_upload_loop_max_backoff_ms` is not correct config, it should be a `cloud_storage_segment_max_upload_interval_sec`. If you will set this value to 10s the archiver will start new upload every 10s or more frequently (if needed) so this will make upload lag smaller.

Reply:
I updated the verbage per @Lazin 's comments. Please review.